home-work14/test_for_human.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from human import Human
+
+
+def test_human_grow():
+    human = Human("John", 30, "male")
+    human.grow()
+    assert human.age == 31, "Age is not 30"
+
+
+def test_human_change_gender():
+    human = Human("Jane", 25, "female")
+    human.change_gender("male")
+    assert human.gender == "male", "Is not male"
```

Comment:
please write more detailed messages. It will help you in future. 

'Human gender is not as expected after changing'